build/utils-node.js

```diff
@@ -459,8 +459,15 @@ module.exports = {
     });
   },
 
+  hasFileInZip: function(zip, pathInZip) {
+    return zip.file(pathInZip);
+  },
+
   closeZip: function(zip, zipPath) {
-    fs.writeFileSync(zipPath, zip.generate({ type: 'nodebuffer' }));
+    fs.writeFileSync(zipPath, zip.generate({
+      type: 'nodebuffer',
+      platform: process.platform
```

Comment:
I guess you refer to below link.
https://github.com/Stuk/jszip/issues/194
It's no written in official document, so I append the link here.